frontend/src/modules/layout/components/paywall-modal.vue

```diff
@@ -70,24 +77,42 @@ const model = computed({
 })
 
 const modal = computed(() => modalContent[props.module])
+const computedFeaturePlan = computed(() => {
+  return config.isCommunityVersion ? 'Custom' : 'Growth'
+})
 
 const modalContent = {
   organizations: {
     title: 'Organizations',
-    preTitle: 'Growth feature',
+    preTitle: `${computedFeaturePlan.value} plan`,
     imageSrc: '/images/paywall/organizations.png',
+    imageClass: 'ml-10 mt-6',
+    imageWrapperClass: 'h-52',
     content:
-      'Get a complete organization directory that you can search, filter, and sort instantly. Each organization also has its own profile page, which highlights key information about that organization and all the community members that belong to it'
+      'Get a complete organization directory that you can search, filter, and sort instantly. Each organization also has its own profile page, which highlights key information about that organization and all the community members that belong to it',
+    knowMore: true
   },
-  // TODO: Community Help Center paywall
-  'community-help-center': {}
+  communityHelpCenter: {
+    title: 'Custom domain',
+    preTitle: `Community Help Center`,
+    imageSrc: '/images/paywall/community-help-center.png',
+    imageClass: 'absolute bottom-0 right-0',
+    imageWrapperClass: 'h-30',
+    content:
+      'In order to setup a custom domain to your help center public page its required to upgrade your plan',
```

Comment:
This sentence feels a bit weird. Probably missing a comma before "its", and shouldn't its be it's?

Reply:
Thanks for noticing this, I blindly copied the copy from Figma 😅